src/quick-lint-js/parse.h

```diff
@@ -3037,7 +3037,6 @@ class parser {
 
```

Comment:
I think a proper fix would involve checking `this->lexer_.has_leading_newline` on line 3034. I could be wrong though.